inc/3rd-party/plugins/ecommerce/woocommerce-multilingual.php

```diff
@@ -5,4 +5,111 @@
 // Compatibility with the currency switcher in WooCommerce Multilingual plugin.
 if ( defined( 'WCML_VERSION' ) ) :
 	add_action( 'wcml_switch_currency', 'rocket_clean_domain' );
```

Comment:
This action should be removed, it's no longer necessary now with the new cookie-compatibility code added

Reply:
Done @Tabrisrp :)